lorawan-device/src/mac/mod.rs

```diff
@@ -223,6 +228,27 @@ impl Mac {
         }
     }
 
+    /// Handles a received RF frame during RXC window. Returns None is unparseable, fails decryption,
```

Comment:
Returns `None` if frame is unparseable or fails either decryption or MIC verification. (And I don't really understand the success case...).

Reply:
In English, saying `A, B, or C` is equivalent to saying `A||B||C`. I think it's fine as is on that end?

However, I've tried to clarify the success case better.

Reply:
The **is** after None should be 'if'?

Reply:
Oh right on about that! I thought you were emphasizing the ORs - thanks